kaznet/apps/ona/api.py

```diff
@@ -295,6 +322,21 @@ def fetch_form_data(  # pylint: disable=too-many-arguments
     return request(url=url, method='GET', args=query_params)
 
 
+def sync_submission_review(instance_id, ona_review_status, comment):
+    """
+    ensure Submission is in sync with Submission in onadata.
+    """
+    args = {'note': comment, "status": ona_review_status,
+            "instance": instance_id}
+    instance = Instance.objects.get(ona_pk=instance_id)
```

Comment:
We need to have a try here, to prevent exceptions in case `instance_id` does not exist in the DB.